cmd/ctrl.go

```diff
@@ -126,19 +143,91 @@ func newQueryChangefeedCommand() *cobra.Command {
 		Use:   "query",
 		Short: "Query information and status of a replicaiton task (changefeed)",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			info, err := cdcEtcdCli.GetChangeFeedInfo(context.Background(), *changefeedID)
+			info, err := cdcEtcdCli.GetChangeFeedInfo(context.Background(), changefeedID)
+			if err != nil && errors.Cause(err) != model.ErrChangeFeedNotExists {
```

Comment:
```suggestion
			if err != nil {
```
we can just return error if `ChangeFeedStatus` does not exist, as the  `ChangeFeedStatus` is always put in etcd when a changefeed is created.